workspaces/ui-v2/src/components/navigation/TopNavigation.tsx

```diff
@@ -72,6 +90,10 @@ const useStyles = makeStyles((theme) => ({
     position: 'fixed',
     width: '100%',
   },
+  spacing: {
+    height: 41, //40px height + 1px border
```

Comment:
Having a class named "spacing" smells of css hackery. Haven't actually dug into how this is all structured, but 98/100 times it's an indicator of layout needing to be structured differently.